### PR TITLE
Feature/add qa selector

### PR DESCRIPTION
**Pull request by strange-developer**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+  "name": "test-utilities",
```
Comment (hvolschenk) [inline package.json]: Very nice, it is still available: https://www.npmjs.com/package/test-utilities

File: package.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+  "version": "1.0.0",
```
Comment (hvolschenk) [inline package.json]: Are you not going to use something like `semantic-release`?

If so, this should be `0.0.0-semver`.

File: src/qa-selector/index.js [hunk 3]
```diff
@@ -0,0 +1 @@
+export default (qaTag) => `[data-qa="${qaTag}"]`;
```
Comment (hvolschenk) [inline src/qa-selector/index.js]: Does `eslint` not moan about the brackets around `qaTag`?

Should it not be:
```
export default qaTag => `[data-qa="${qaTag}"]`;
```

**Comment by hvolschenk** [comment]
@strange-developer I also think it'd be useful to add a **travis-ci** build to this so it runs on MRs?

Maybe also one for **coveralls**.

File: .babelrc [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "presets": [
+    "env"
+  ]
+}
```
Comment (hvolschenk) [inline .babelrc]: Doesn't your editor add newlines for you?

Comment (strange-developer) [reply .babelrc]: Not on dot files. Perhaps I could configure that though.

Comment (strange-developer) [reply .babelrc]: Fixed

File: package.json (hunk 2 above)
Comment (strange-developer) [reply package.json]: Not so sure about `semantic-release` but I'm definitely going to add a `travis` build to run on PR's

**Comment by strange-developer** [comment]
I've added Docker as well. Service name is `app`.

File: Dockerfile [hunk 5]
```diff
@@ -0,0 +1,5 @@
+FROM node:carbon-alpine
```
Comment (hvolschenk) [inline Dockerfile]: You might want to swap this to `node:alpine` so you can use `npm audit`, or do you want to wait for a named LTS Docker image?

File: .babelrc (hunk 4 above)
Comment (hvolschenk) [reply .babelrc]: My diff still shows no newline, same thing in the `.eslintrc`, `.npmignore`, `Dockerfile` and `docker-compose.yml` files. You sure your setting is working :stuck_out_tongue:

File: package.json (hunk 2 above)
Comment (hvolschenk) [reply package.json]: Okay wow, just noticed that I cannot mark comments as resolved on _GitHub_. I guess I'll just add a "Resolved" comment then.

Resolved.

File: package.json (hunk 1 above)
Comment (hvolschenk) [reply package.json]: Resolved.

**Comment by strange-developer** [comment]
Coveralls have been applied @hvolschenk

**Pull request merged**